packages/rum-core/src/performance-monitoring/navigation/capture-navigation.js

```diff
@@ -42,6 +42,12 @@ function captureNavigation(transaction) {
    * transactions set this flag to true
    */
   if (!transaction.captureTimings) {
+    // Make sure page load transactions always starts at 0 irrespective if we capture navigation metrics or not
+    // otherwise we would be reporting different page load durations for sampled and unsampled transactions
+    if (transaction.type === PAGE_LOAD) {
```

Comment:
I didn't merge that logic, because we use transaction._start in the captureTimings logic (before initializing to 0)

```
    if (transaction.marks && transaction.marks.custom) {
      const customMarks = transaction.marks.custom
      Object.keys(customMarks).forEach(key => {
        customMarks[key] += transaction._start
      })
    }
```

Reply:
Wow, TIL